src/components/calculators/Payslip.vue

```diff
@@ -90,9 +90,18 @@ Payslip calculator.
             <p>You will be covered under {{ studentLoanPlan }}</p>
           </fieldset>
         </div>
-        <!--</fieldset>-->
       </fieldset>
-      <!--</fieldset>-->
+      <fieldset class="form-item">
+        <p class="form-item__title">Tax year:</p>
+        <div class="form-item__wrapper form-item__wrapper--year-selector  flex-wrap">
+          <label class="form-item__label flex-grow" :class="{active: input.date === '2018'}">
```

Comment:
Might be better to go in reverse date order here so 2019 is first as the default